### PR TITLE
feat(release): Prepare 1.11.0 release with Nextcloud 28 support

### PR DESCRIPTION
**Pull request by mejo-**

### CODE REVIEW

**Comment by nickvergessen** [comment]
- [ ] `@nextcloud/dialogs` should be updated to 5
- [ ] `@nextcloud/vue` should be updated to 8

**Comment by mejo-** [comment]
Good point @nickvergessen. Did both now in this PR.

**Comment by mejo-** [comment]
Ok, well. Seems like we have to migrate from `NcMultiselect` to `NcSelect` first :disappointed:

File: appinfo/info.xml [hunk 1]
```diff
@@ -27,7 +27,7 @@
 	<screenshot>https://raw.githubusercontent.com/nextcloud/user_retention/main/docs/screenshot.png</screenshot>
 
 	<dependencies>
-		<nextcloud min-version="25" max-version="27" />
+		<nextcloud min-version="25" max-version="28" />
```
Comment (nickvergessen) [inline appinfo/info.xml:30]: let's branch off the previous version, so the UI look is aligned with the rest of the server?
```suggestion
		<nextcloud min-version="28" max-version="28" />
```
Or at least drop the EOL version:
```suggestion
		<nextcloud min-version="26" max-version="28" />
```

Comment (mejo-) [reply appinfo/info.xml:30]: I don't think it's worth doing this. The success/error dialogs didn't change with `@nextcloud/dialogs` 5. Also I just tested this branch on `stable27` and it works flawlessly. I agree with dropping 25 support though :+1:

**Comment by mejo-** [comment]
Requires https://github.com/nextcloud/user_retention/pull/629 for green CI.

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**